frame/dapps-staking/src/tests.rs

```diff
@@ -8,7 +8,9 @@ use testing_utils::*;
 #[test]
 fn bond_and_stake_different_eras_is_ok() {
     ExternalityBuilder::build().execute_with(|| {
-        run_to_block(1);
+        // prepare initial era
+        mock::prepare_era_setup();
```

Comment:
I don't think you need the 'mock::' part.

Also, the comment part is a bit redundant. 
If method name isn't explanatory enough, we should rename it to 'prepare_initial_era()'